packages/ses/test/test-sloppy-globals-scope-terminator.js

```diff
@@ -0,0 +1,87 @@
+import test from 'ava';
+import { createSloppyGlobalsScopeTerminator } from '../src/sloppy-globals-scope-terminator.js';
+
+test('sloppyGlobalsScopeTerminator/get - always has properties but they are undefined', t => {
+  t.plan(4);
+
+  const globalObject = {};
+  const scopeTerminator = createSloppyGlobalsScopeTerminator(globalObject);
+
+  t.is(Reflect.has(scopeTerminator, 'eval'), true);
+  t.is(Reflect.get(scopeTerminator, 'eval'), undefined);
+
+  t.is(Reflect.has(scopeTerminator, 'xyz'), true);
+  t.is(Reflect.get(scopeTerminator, 'xyz'), undefined);
```

Comment:
Can we add a test for another prop `abc` (pick a name) that exists on the `globalObject`, so it's showing that `scopeTerminator` doesn't leak it?

I guess that behavior is tested in the `set` test below, but it'd be good to have an explicit `get` only test.